.github/workflows/build-docker-artifact.yaml

```diff
@@ -27,17 +40,17 @@ jobs:
           password: ${{ secrets.GITHUB_TOKEN }}
       - name: Build Docker image
         run: |
-          ./scripts/docker/build_docker_image.sh ${{ env.TT_METAL_DOCKER_IMAGE }}:${{ env.TT_METAL_DOCKER_IMAGE_TAG }}
+          ./scripts/docker/build_docker_image.sh ${{ inputs.os }}
       - name: Determine Docker Image Tag
         id: set-image
         run: |
           if [[ "${GITHUB_REF_NAME}" == "main" ]]; then
             echo "IMAGE_TAG=latest" >> $GITHUB_ENV
           else
-            echo "IMAGE_TAG=dev-${GITHUB_ACTOR}" >> $GITHUB_ENV
+            echo "IMAGE_TAG=dev-${GITHUB_REF_NAME//\//-}" >> $GITHUB_ENV
```

Comment:
So things like `mchiou/branch-1` would turn into `mchiou-branch-1`?

Reply:
yes, Docker tags do not allow '/' in tags. It will throw an error if a '/' is in the tag